Sources/DatabaseKit/Database/Database.swift

```diff
@@ -6,13 +8,15 @@ public protocol Database {
     /// static methods on this type that supplied an instance of `Connection`.
     associatedtype Connection: DatabaseConnection
 
+    var eventLoop: EventLoop { get }
+    
     /// Creates a new `DatabaseConnection` that will perform async work on the supplied `Worker`.
     ///
     ///     let conn = try database.newConnection(on: ...).wait()
     ///
     /// - parameters:
     ///     - worker: `Worker` to perform async work on.
-    func newConnection(on worker: Worker) -> Future<Connection>
+    func newConnection() -> EventLoopFuture<Connection>
```

Comment:
Shouldn't this be `makeConnection()` as per the [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage)?

Reply:
@Mordil yeah, I think it should. I've added this to the TODO, thanks.